### ISSUE TITLE
memoryMetric fails with atrace.

### ISSUE THREAD

**Issue by benshayden** [catapult-project/catapult#3795]
[This google-only trace](https://drive.google.com/a/google.com/file/d/0B-IDmAzWp11iVnQ1NTctUU9oZk0) contains both atrace and memory dumps. memoryMetric throws `Unknown process name "/init". Please add it to |CHROME_PROCESS_NAMES|.`
Could either memoryMetric filter processes or canonicalizeProcessName return UNKNOWN instead of requiring all processes to be known?

**Comment by chromy** [comment]
Yeah sgtm, I actually saw another case of this the other day in a slightly different context. I'm out on Monday (public holiday) but I'll take a look on Tuesday, hopefully should be a simple fix.

**Comment by chromy** [comment]
Should be fixed!
